js/web/lib/wasm/binding/ort-wasm.d.ts

```diff
@@ -102,6 +102,9 @@ export interface OrtWasmModule extends EmscriptenModule {
   _OrtTrainingCopyParametersFromBuffer?
       (trainingHandle: number, parametersBuffer: number, parameterCount: number, trainableOnly: boolean): number;
 
+  _OrtTrainingGetInputOutputCount?(sessionHandle: number, inputCountOffset: number, outputCountOffset: number): number;
+  _OrtTrainingGetInputOutputName?(sessionHandle: number, index: number, isInput: boolean): number;
```

Comment:
nit:
```suggestion
  _OrtTrainingGetInputOutputName?(trainingHandle: number, index: number, isInput: boolean): number;
```

Reply:
applied this and the previous naming suggestion in the last commit